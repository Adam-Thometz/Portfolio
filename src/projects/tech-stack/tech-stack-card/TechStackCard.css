.TechStackCard {
  height: 75px;
  width: 75px;
  display: inline-block;
  text-align: center;
  padding: 15px;
  margin: 5px;
  border-radius: 25px;
}

.TechStackCard-img-wrapper img {
  background-color: rgba(245, 245, 245, 0.833);
  height: 60px;
  width: 60px;
  box-shadow: none;
  border: none;
}

.TechStackCard-caption {
  font-size: .9rem;
  color: #332c2c;
}

@media (max-width: 1200px) {
  .TechStackCard {
    height: 40px;
    width: 40px;
  }
  
  .TechStackCard-img-wrapper img {
    height: 25px;
    width: 25px;
  }
}