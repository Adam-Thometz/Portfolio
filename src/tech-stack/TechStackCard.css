.TechStackCard {
  background-color: rgba(245, 245, 245, 0.466);
  height: 85px;
  width: 85px;
  display: inline-block;
  text-align: center;
  padding: 15px;
  margin: 5px;
  border-radius: 25px;
}

.TechStackCard-img-wrapper img {
  height: 70px;
  width: 70px;
}

.TechStackCard-caption {
  font-size: .9rem;
  color: #332c2c;
}