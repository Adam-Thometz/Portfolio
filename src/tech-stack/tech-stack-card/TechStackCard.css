.TechStackCard {
  height: 100px;
  width: 70px;
  display: inline-block;
  text-align: center;
  padding: 10px;
  margin: 5px;
  border-radius: 25px;
}

.TechStackCard-img {
  background-color: rgba(245, 245, 245, 0.833);
  height: 60px;
  width: 60px;
  box-shadow: none;
  border: none;
}

.TechStackCard-caption {
  font-size: .9rem;
  color: #eeeeee;
  text-align: center;
}