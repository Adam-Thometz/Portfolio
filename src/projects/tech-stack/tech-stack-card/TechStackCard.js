import React from "react";
import './TechStackCard.css';

const TechStackCard = ({ name, icon }) => {
  return (
    <div className="TechStackCard">
      <div className="TechStackCard-img-wrapper">
        <img src={icon} alt={name} />
      </div>
      <span className="TechStackCard-caption">{name}</span>
    </div>
  );
};

export default TechStackCard;