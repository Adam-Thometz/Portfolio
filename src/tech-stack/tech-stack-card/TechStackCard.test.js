import React from "react";
import { render } from "@testing-library/react";
import TechStackCard from "./TechStackCard";

import jsIcon from "../../_media/tech-stack-icons/javascript.png";

describe("TechStackCard component", () => {
  it("renders without crashing", () => {
    render(<TechStackCard name="Javascript" icon={jsIcon} />);
  });

  it("matches the snapshot", () => {
    const { asFragment } = render(
      <TechStackCard name="Javascript" icon={jsIcon} />
    );
    expect(asFragment()).toMatchSnapshot();
  });
});
