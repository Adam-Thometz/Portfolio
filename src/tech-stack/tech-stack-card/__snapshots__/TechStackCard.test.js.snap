// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TechStackCard component matches the snapshot 1`] = `
<DocumentFragment>
  <figure
    class="TechStackCard"
  >
    <img
      alt="Javascript"
      class="TechStackCard-img"
      src="javascript.png"
    />
    <figcaption
      class="TechStackCard-caption"
    >
      Javascript
    </figcaption>
  </figure>
</DocumentFragment>
`;
